### DISCUSSION TITLE
Adding Javascript and CSS to Docsy

### DISCUSSION THREAD

**Discussion by jmichelgarcia** [google/docsy#1228] [Q&A] [answered]
Hi,

I wanted to include some custom JS code to Docsy, currently I'm doing so by extending `head-end.html` partial.
Is there any best practice with Docsy? I couldn't find anything on the website.

TIA.

**Comment by fekete-robert** [comment, marked as answer]
Yes, that's the way to do it: https://www.docsy.dev/docs/adding-content/lookandfeel/#customizing-templates

**Reply by jmichelgarcia** [reply]
Thank you! Somehow I missed that in the docs!
